es-space-mgmt/main/default/lwc/customerList/__tests__/data/getCustomerList.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
Update your test data to use proper field names with capitals:
```
Id, Email, Name, City, State, Status
```

Reply:
The wrapper class returns data with lower case field names, and the component also refers to these fields using lower case only.. So if I change the test data, the tests will fail.

Reply:
Damn, I really thought that these were record fields. Good catch.